packages/amplication-data-service-generator/src/server/static/src/auth/UserInfo.ts

```diff
@@ -5,8 +5,8 @@ import { User } from "../user/user";
 
 @ObjectType()
 export class UserInfo implements Partial<User> {
-  @Field(() => String)
-  id!: string;
+  @Field(() => String || Number)
+  id!: string | number;
```

Comment:
This should be generated dynamically by the type of the ID field on the User entity (either a string or a number)